core/snippets/snippets_insert.py

```diff
@@ -47,3 +47,28 @@ def insert_snippet_by_name_with_phrase(name: str, phrase: str):
             )
 
         actions.user.insert_snippet_by_name(name, substitutions)
+
+    def insert_wrapper_snippet(
+        body: str,
+        target: Any,
```

Comment:
I don't see the relevant import for Any fyi

Reply:
Done!